documentation/book/api/overview.adoc

```diff
@@ -11,25 +8,6 @@ The Strimzi HTTP Bridge for Apache Kafka provides a REST API for integrating HTT
 __Version__ : 0.1.0
 
 
-=== Contact information
-[%hardbreaks]
-__Contact Email__ : cncf-strimzi-dev@lists.cncf.io
-
-
-=== License information
-[%hardbreaks]
-__License__ : Apache 2.0
-__License URL__ : http://www.apache.org/licenses/LICENSE-2.0.html
-__Terms of service__ : http://swagger.io/terms/
-
-
-=== URI scheme
-[%hardbreaks]
-__Host__ : bridge.swagger.io
-__BasePath__ : /
-__Schemes__ : HTTP
-
-
```

Comment:
@ShubhamRwt -- I took this out of the API reference. Can we exclude from bridge API build and just have it in the readme?

Reply:
Yes I guess we can remove it and mention it in the Readme